src/components/SharingSidebarView.vue

```diff
@@ -185,6 +186,7 @@ import NcSelect from '@nextcloud/vue/dist/Components/NcSelect.js'
 import Tooltip from '@nextcloud/vue/dist/Directives/Tooltip.js'
 import Plus from 'vue-material-design-icons/Plus.vue'
 import Close from 'vue-material-design-icons/Close.vue'
+import logger from '../services/logger.ts'
```

Comment:
Can we sort the import block. :see_no_evil:

Reply:
Use `Volar` sorting here now